### ISSUE TITLE
Multiline regex should use non-greedy match

### ISSUE THREAD

**Issue by w2qiao** [jeanralphaviles/comment_parser#5]
https://github.com/jeanralphaviles/comment_parser/blob/f57fa777a0e4b814c1ff2119a52a9e24f86ab2a1/comment_parser/parsers/c_parser.py#L42

when parsing 
`/*
abc
*/
/*
abc
*/`

output is 
`abc
*/
/*
abc`

 (?P<multi> /\*(?P<multi_content>(.|\n)+?)?\*/)  works in this case. See https://stackoverflow.com/questions/14213848/difference-between-and

**Comment by jeanralphaviles** [comment]
I couldn't reproduce your bug, do you have more context?

```python
>>> from comment_parser import comment_parser
>>> comment_parser.extract_comments('/tmp/test.c')
[Comment( abc , 3, True), Comment( 123 , 3, True)]
```

*/tmp/test.c*
```c
#include <stdio.h>

int main() {
   /* abc */ /* 123 */
   printf("Hello, World!");
   return 0;
}
```

I've added a test case for the C parser for your example. If you find another way to reproduce the bug please reopen this issue.

**Comment by jeanralphaviles** [comment]
Oops, it seems that this was fixed in #8 and was never tied to this issue.
